.github/workflows/cypress.yaml

```diff
@@ -154,23 +138,23 @@ jobs:
               run: |
                   echo "CYPRESS_RUST_CRYPTO=1" >> "$GITHUB_ENV"
 
-            - name: Run Cypress tests
+            - name: Run Cypress tests via knapsack pro
               uses: cypress-io/github-action@ebe8b24c4428922d0f793a5c4c96853a633180e3 # v6.6.0
               with:
                   working-directory: matrix-react-sdk
-                  # The built-in Electron runner seems to grind to a halt trying to run the tests, so use chrome.
-                  browser: ${{ steps.setup-chrome.outputs.chrome-path }}
                   headed: true
                   start: npx serve -p 8080 -L ../webapp
                   wait-on: "http://localhost:8080"
-                  record: true
+                  record: false
                   parallel: false
-                  command-prefix: "yarn percy exec --parallel --"
-                  ci-build-id: ${{ needs.prepare.outputs.uuid }}-${{ matrix.crypto }}
-                  spec: cypress/e2e/[${{ matrix.segment }}]*/**
+                  # The built-in Electron runner seems to grind to a halt trying to run the tests, so use chrome.
+                  command: npx knapsack-pro-cypress --config trashAssetsBeforeRuns=false --browser ${{ steps.setup-chrome.outputs.chrome-path }}
```

Comment:
We shouldn't be losing the percy bit here otherwise it makes re-enabling not a case of reverting the disabling PR

Reply:
The percy command or the percy variables that I accidentally removed?

Reply:
If we're talking about the command, `yarn percy exec --parallel -- npx knapsack-pro-cypress --config trashAssetsBeforeRuns=false --browser ${{ steps.setup-chrome.outputs.chrome-path }}` does not seem to work, see https://github.com/MidhunSureshR/matrix-react-sdk/actions/runs/6891941393/job/18748139242#step:7:122

Reply:
The command, so that this change isn't touching percy at all and merely swapping out the parallisation mechanism please

Reply:
I'm not seeing any documentation for knapsack-pro-cypress & percy, so adapting knapsack might mean killing Percy rather than just temporarily disabling it

Reply:
The test command actually ran is `yarn percy exec --parallel -- npx knapsack-pro-cypress --config trashAssetsBeforeRuns=false --browser` i.e no path follows --browser ...

Reply:
Probably because I'm not using the github_env anymore

Reply:
works fine now: https://github.com/MidhunSureshR/matrix-react-sdk/actions/runs/6892362099/job/18749476893